### ISSUE TITLE
Feature request: run before.update only once with multi: true

### ISSUE THREAD

**Issue by davidworkman9** [matb33/meteor-collection-hooks#106]
I'd love to be able to run a ```before.update``` hook only once for a ```{ multi: true }``` update.

Use case:
I change the modifier to set two fields on one of my collections, ```lastModifiedBy``` and ```lastModifiedDate```, but it runs for every affected document, which is unnecessary, I don't need to know what any of the documents look like, just need to be sure that they get their last modified timestamp and userId updated.

**Comment by mizzao** [comment]
Yeah, this is a side effect of the `update` hooks not taking place as a single function call. I think we were talking about this in another issue as well, but can't find that at the moment.

**Comment by matb33** [comment]
You could try debouncing it, either manually with a setTimeout 0 that you just cancel on each iteration:
```
Meteor.clearTimeout(t);
t = Meteor.setTimeout(function () {
  // do that update to your other collection for lastModifiedBy and lastModifiedDate
}, 0);
```

Or use `_.debounce` with delay of 0 -- just to group all those calls into a single one that runs on next tick.

This of course will only work if all those updates fire during the same tick, which I assume they do (fingers crossed)
